pennylane/qnn/keras.py

```diff
@@ -52,6 +54,10 @@ class KerasLayer(Layer):
             arguments of the `add_weight()
             <https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer#add_weight>`__
             method and values being the corresponding specification.
+        argnum (Union[Sequence[int], int]): Argument location of the non-trainable inputs for
+            the circuit. This allows batch execution by creating executable circuits for each
+            input example with the same trainable weights. Default ``None``.
+            See :class:`~.pennylane.transforms.batch_input` for more details.
```

Comment:
Nice! If the default is None, does that mean no batching occurs?

I wonder if it is working making this more descriptive, e.g., `batch_idx` rather than `argnum` 🤔 What are your thoughts?

Reply:
I totally agree with `batch_idx` but the rest of the program uses `argnum` that's why I choose that. If you think that's okay to change, fine for me.

If none, the class basically does what it was doing before, for backwards compatibility.